### PR TITLE
MINOR: Cleanup in tests to avoid threads being left behind

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

File: core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala [hunk 1]
```diff
@@ -119,6 +119,7 @@ class ResetConsumerGroupOffsetTest extends KafkaServerTestHarness {
     printConsumerGroup()
 
     AdminUtils.deleteTopic(zkUtils, topic1)
+    consumerGroupCommand1.close()
```
Comment (ijuma) [inline core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala]: It looks like `new ConsumerGroupExecutor(brokerList, 1, group, topic1)` is not shutdown. Would it make sense to have a factory method that ensures that everything is shutdown in a tearDown method like we do elsewhere?

File: core/src/test/scala/unit/kafka/controller/ControllerEventManagerTest.scala [hunk 2]
```diff
@@ -33,6 +33,7 @@ class ControllerEventManagerTest {
 
   private var controllerEventManager: ControllerEventManager = _
 
+  @After
```
Comment (ijuma) [inline core/src/test/scala/unit/kafka/controller/ControllerEventManagerTest.scala:36]: Nice catch, my bad for missing this.

**Review by ijuma — COMMENTED** [review]
Thanks for the PR, @rajinisivaram. Left a comment with regards to `ResetConsumerGroupOffsetTest`.

File: core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala (hunk 1 above)
Comment (rajinisivaram) [reply core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala]: @ijuma Thank you for the review. Added `tearDown`.

**Review by ijuma — APPROVED** [review]
Thanks, LGTM if the tests pass.

**Pull request closed without merging**